### PR TITLE
Update Android Deps

### PR DESCRIPTION
**Pull request by Ishan09811**
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/7383)

### CODE REVIEW

File: src/android/app/build.gradle.kts [hunk 1]
```diff
@@ -173,21 +173,21 @@ android {
...
-    implementation("androidx.activity:activity-ktx:1.8.0")
+    implementation("androidx.activity:activity-ktx:1.8.2")
     implementation("androidx.fragment:fragment-ktx:1.6.2")
     implementation("androidx.appcompat:appcompat:1.6.1")
     implementation("androidx.documentfile:documentfile:1.0.1")
-    implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.1")
+    implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.2")
     implementation("androidx.slidingpanelayout:slidingpanelayout:1.2.0")
-    implementation("com.google.android.material:material:1.9.0")
+    implementation("com.google.android.material:material:1.10.0")
```
Comment (t895) [inline src/android/app/build.gradle.kts:182]: Please revert this change. There's a bug in v1.10.0 that breaks insets for the navigation rail.

Comment (Ishan09811) [reply src/android/app/build.gradle.kts:182]: > Please revert this change. There's a bug in v1.10.0 that breaks insets for the navigation rail. For context, I have a pull request in the material components github page with a fix that's still pending. [material-components/material-components-android#3991](https://github.com/material-components/material-components-android/pull/3991)

ok

**Comment by t895** [comment]
Also why did you update the dependencies that you did? It doesn't seem like you just updated everything since there are a few in there that have updates available that you didn't change and viewmodel-ktx has a newer version than 2.6.2.

**Comment by Ishan09811** [comment]
@t895 
ok I will update it to letest 2.7.0 stable version and I will also check the other deps are upto date with letest stable versions.

**Comment by t895** [comment]
You also need to make sure to update the navigation safe args plugin in the top-level build.gradle file

**Comment by Ishan09811** [comment]
@t895 
done

**Review by t895 — APPROVED** [review]
